addons/l10n_hu_edi/models/res_partner.py

```diff
@@ -13,6 +13,26 @@ class ResPartner(models.Model):
         index=True,
     )
 
+    def _compute_is_company(self):
+        """
+            A partner can be identified as a company in two ways:
+
+            1. VAT follows the European format:
+            - Starts with 'HU' and Followed by 8 digits
+            Example: HU12345678
+
+            2. VAT follows the Hungarian domestic format:
+            - 8 digits - [2/4/5] - 2 digits
+            Example: 12345678-2-43
+        """
```

Comment:
We could just take Hungarian partners, and below do (partner.vat or '').replace()..., that way we handle all hungarian people in this override (and let other override deal with only other partners). That way the compute is self contained.